Rubberduck.CodeAnalysis/Inspections/Abstract/IdentifierReferenceInspectionBase.cs

```diff
@@ -16,13 +16,72 @@ protected IdentifierReferenceInspectionBase(RubberduckParserState state)
         {}
 
         protected abstract bool IsResultReference(IdentifierReference reference, DeclarationFinder finder);
-        protected abstract string ResultDescription(IdentifierReference reference, dynamic properties = null);
+        protected abstract string ResultDescription(IdentifierReference reference);
 
-        protected virtual (bool isResult, object properties) IsResultReferenceWithAdditionalProperties(IdentifierReference reference, DeclarationFinder finder)
+        protected virtual ICollection<string> DisabledQuickFixes(IdentifierReference reference) => new List<string>();
+
+        protected override IEnumerable<IInspectionResult> DoGetInspectionResults()
         {
-            return (IsResultReference(reference, finder), null);
+            var finder = DeclarationFinderProvider.DeclarationFinder;
+
+            var results = new List<IInspectionResult>();
+            foreach (var moduleDeclaration in State.DeclarationFinder.UserDeclarations(DeclarationType.Module))
+            {
+                if (moduleDeclaration == null)
+                {
+                    continue;
+                }
+
+                var module = moduleDeclaration.QualifiedModuleName;
+                results.AddRange(DoGetInspectionResults(module, finder));
+            }
+
```

Comment:
The other inspection base classes also iterated over all user defined Projects. Why does this not?

Reply:
This base class is for identifier references inside modules. Since projects do not contain code directly, there cannot be identifier references in it.

The other base classes deal with declarations and projects have a project declaration, which is the target of some inspections.